Frontend/Gmfctn/src/app/shared/models/user-short-info.ts

```diff
@@ -0,0 +1,3 @@
+import { User } from "./user";
+
+export type UserSI = Pick<User, 'id' | 'firstName' | 'lastName' | 'xp' | 'avatarId'>
```

Comment:
New line et the end of the file.
https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/

Reply:
Read.
Started ng lint and fixed others issues